Source/items.cpp

```diff
@@ -2534,6 +2534,19 @@ void RecreateEar(int ii, WORD ic, int iseed, int Id, int dur, int mdur, int ch,
 }
 
 #ifdef HELLFIRE
+void items_427A72()
+{
+	PkItemStruct id;
+	if (dword_691CB0) {
+		if (CornerItemMaybe.IDidx >= 0) {
+			PackItem(&id, &CornerItemMaybe);
+			SRegSaveData("Hellfire", off_4A5AC4, 0, (BYTE *)&id, 19);
+		} else {
+			SRegSaveData("Hellfire", off_4A5AC4, 0, (BYTE *)(0xDEAFB0B), 1); //TODO: some address should be here, figure out what

```

Comment:
```suggestion
			SRegSaveData("Hellfire", off_4A5AC4, 0, (BYTE *)"", 1);
```
Your wellcome

Reply:
wtf :p

Reply:
The compiler deduplicates strings, this one appears a lot ...